src/main/java/SYSC4806Project/ItemQuantityList.java

```diff
@@ -57,7 +60,7 @@ public boolean removeItems(Product product, int quantity) {
                     System.out.println("Attempted to remove more quantity than available. None were removed.");
                     return false;
                 }
-                itemQuantityPair.setQuantity(quantity - itemQuantityPair.getQuantity());
+                itemQuantityPair.setQuantity(itemQuantityPair.getQuantity() - quantity);
```

Comment:
@Rebecca-Elliott Just switched the subtraction order to "existing quantity - quantity to remove". Can you check this makes sense? Thanks!

Reply:
correct.